### PR TITLE
Github action update for SPAship manager

### PR DESCRIPTION
**Pull request by arkaprovob**
## Resolves

GitHub action for the manager is updated as per build requirements.

## Explain the feature/fix

Only change in manager-build action

## Does this PR introduce a breaking change

No

## Screenshot(s)
Screenshot from 2022-09-20 20-39-40
Screenshot from 2022-09-20 20-39-35

### CODE REVIEW

**Review by akhilmhdh — APPROVED** [review]

**Review by soumyadip007 — APPROVED** [review]

File: .github/workflows/manager-ci.yaml [hunk 1]
```diff
@@ -37,14 +40,65 @@ jobs:
...
+        run: echo ${{ steps.docker_build_qa.outputs.digest }}
+
+
+      - name: Stage Build and push into repository
+        if: github.event.inputs.env == 'Stage'
+        id: docker_build_stage
+        uses: docker/build-push-action@v2
+        with:
+          context: ./packages/manager
+          #file: ./packages/manager/Dockerfile
```
Comment (riginoommen) [inline .github/workflows/manager-ci.yaml:81]:
```suggestion
```

File: .github/workflows/manager-ci.yaml [hunk 2]
```diff
@@ -37,14 +40,65 @@ jobs:
...
+        run: echo ${{ steps.docker_build_dev.outputs.digest }}
+
+
+      - name: Qa Build and push into repository
+        if: github.event.inputs.env == 'Qa'
+        id: docker_build_qa
+        uses: docker/build-push-action@v2
+        with:
+          context: ./packages/manager
+          #file: ./packages/manager/Dockerfile
```
Comment (riginoommen) [inline .github/workflows/manager-ci.yaml:65]:
```suggestion
```

File: .github/workflows/manager-ci.yaml [hunk 3]
```diff
@@ -37,14 +40,65 @@ jobs:
...
 
-      - name: Build and push into repository
-        id: docker_build
+      - name: Dev Build and push into repository
+        if: github.event.inputs.env == 'Dev'
+        id: docker_build_dev
+        uses: docker/build-push-action@v2
+        with:
+          context: ./packages/manager
+          #file: ./packages/manager/Dockerfile
```
Comment (riginoommen) [inline .github/workflows/manager-ci.yaml:49]:
```suggestion
```

File: .github/workflows/manager-ci.yaml [hunk 4]
```diff
@@ -37,14 +40,65 @@ jobs:
...
+        run: echo ${{ steps.docker_build_stage.outputs.digest }}
+
+
+      - name: Prod Build and push into repository
+        if: github.event.inputs.env == 'Prod'
+        id: docker_build_prod
         uses: docker/build-push-action@v2
         with:
           context: ./packages/manager
           #file: ./packages/manager/Dockerfile
```
Comment (riginoommen) [inline .github/workflows/manager-ci.yaml:97]:
```suggestion
```

Comment (riginoommen) [inline .github/workflows/manager-ci.yaml:97]:
```suggestion
```

**Review by riginoommen — CHANGES_REQUESTED** [review]

File: .github/workflows/manager-ci.yaml (hunk 1 above)
Comment (arkaprovob) [reply .github/workflows/manager-ci.yaml]: removed

File: .github/workflows/manager-ci.yaml (hunk 2 above)
Comment (arkaprovob) [reply .github/workflows/manager-ci.yaml]: remoevd

File: .github/workflows/manager-ci.yaml (hunk 3 above)
Comment (arkaprovob) [reply .github/workflows/manager-ci.yaml]: removed

File: .github/workflows/manager-ci.yaml (hunk 4 above)
Comment (arkaprovob) [reply .github/workflows/manager-ci.yaml]: removed

Comment (arkaprovob) [reply .github/workflows/manager-ci.yaml]: removed

**Review by riginoommen — APPROVED** [review]

**Pull request merged**